File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/save-persons-vetting-status_param.ts


import { ISavePersonsVettingStatus_Param } from './isave-persons-vetting-status_param';

export class SavePersonsVettingStatus_Param implements ISavePersonsVettingStatus_Param {
  
	public PersonsVettingID?: number;
	public VettingStatus: string = "";
	public IsClear?: boolean;
	public IsDeny?: boolean;
	public ModifiedAppUserID?: number;
  
}

File: GTTS/INL.VettingService.Models/SavePersonsVettingStatus_Param.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.VettingService.Models
{
    public class SavePersonsVettingStatus_Param : ISavePersonsVettingStatus_Param
    {
        public long? PersonsVettingID { get; set; }
        public string VettingStatus { get; set; }
        public bool? IsClear { get; set; }
        public bool? IsDeny { get; set; }
        public int? ModifiedAppUserID { get; set; }
    }
}
